### DISCUSSION TITLE
German translations

### DISCUSSION THREAD

**Discussion by Thompson-ongithub** [mytourbook/mytourbook#1207] [General]
I'm not sure if it fits better here or under Issues, so I post it here:

I just realized the following in the German version of MTB:
unter Preferences - People _Resting HR_ is translated as _Erholungspuls_ whilst in the tooltip of a tour it's called _Restpuls_.
I'm not sure if _Ruhepuls_ wouldn't be a better translation at all..

Another translation that I'm not really sure about is the German translation of _flat gradient_. Flache Steigung seems the literal translation, it sounds/feels strange but right up until now I couldn't think of a better one... :(

**Comment by wolfgang-ch** [comment]
I changed the wording to Ruhepuls in https://github.com/mytourbook/mytourbook/commit/d588c34dc5c939eb5b08a36d545163a2ea35a15f, _Ruhepuls_ was already used

![image](https://github.com/mytourbook/mytourbook/assets/1283445/07b53023-9484-4949-9d52-459cdbea6902)

**Reply by Thompson-ongithub** [reply]
Thanks
indeed, there was Ruhepuls, Restpuls and Erholungspuls ;)
